app/code/Magento/Catalog/Test/Unit/Model/Product/Option/SaveHandlerTest.php

```diff
@@ -40,8 +40,9 @@ protected function setUp(): void
     {
         $this->entity = $this->getMockBuilder(Product::class)
             ->disableOriginalConstructor()
+            ->setMethods(['getOptionsSaved', 'getCanSaveCustomOptions', 'getOptions', 'dataHasChangedFor', 'getSku'])
             ->getMock();
-        $this->optionMock = $this->getMockBuilder(Option::class)
+        $this->option = $this->getMockBuilder(Option::class)
```

Comment:
best practice is to have "mock" suffix for cases when object is really mock. Please revert renaming